File: bddtests/src/main/kotlin/mocking/tpp/models/ViewAppointmentsReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement
import javax.xml.bind.annotation.XmlElement

@XmlRootElement(name = "ViewAppointmentsReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class ViewAppointmentsReply (
    @XmlAttribute var patientId: String = "default patientId",
    @XmlAttribute var onlineUserId: String = "default onlineUserId",
    @XmlAttribute var uuid: String = "default uuid",
    @XmlElement var Appointment: List<Appointment> = listOf()
    )

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/Appointments/ViewAppointmentsReply.cs
using System;
using System.Collections.Generic;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models.Appointments
{
    [Serializable]
    public class ViewAppointmentsReply
    {
        [XmlAttribute("patientId")]
        public string PatientId { get; set; }

        [XmlAttribute("onlineUserId")]
        public string OnlineUserId { get; set; }

        [XmlAttribute("uuid")]
        public string UuId { get; set; }

        [XmlElement("Appointment", typeof(Appointment))]
        public List<Appointment> Appointments { get; set; }
    }
}
